Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Base
+FROM python as base
+
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+WORKDIR /code
+
+ENV PATH "/code:/root:/root/.poetry/bin:${PATH}"
+
+# add and install python requirements
+COPY pyproject.toml poetry.lock /code/
+RUN poetry install
+
+# Copy the rest of the code
+COPY / /code/
```

Comment:
If you copy the whole directory, this includes a lot things you don't want. 

Essentially, everything in your .gitignore should be added to a .dockerignore file, along with some more like `.git`, `.gitignore`, `README.md` `Dockerfile`.

The most important thing to add is `.env` - you don't want the built image to contain environment specific values and it should definitely not include secrets.